pandas/core/generic.py

```diff
@@ -3126,6 +3156,16 @@ def to_csv(
 
             .. versionadded:: 1.1.0
 
+        storage_options : dict, optional
```

Comment:
i wonder if there is a way to share doc-strings components for all of these i/o methods

Reply:
I am not aware of a way

Reply:
yeah i think we can do this with our shared docs infra, but out of scope for now